src/Components/Publishing/News/__tests__/NewsPanel.test.tsx

```diff
@@ -19,3 +20,15 @@ it("Links headlines to article", () => {
   expect(wrapper.html()).toMatch(`/news/${NewsArticle.slug}`)
   expect(wrapper.html()).toMatch(`/news/${FeatureArticle.slug}`)
 })
+
+it("Renders the date correctly in the News Panel Header", () => {
+  const wrapper = mount(<NewsPanel articles={[NewsArticle, FeatureArticle]} />)
+  const today = getDate(new Date().toISOString(), "monthDay")
+
+  expect(
+    wrapper
+      .find("#news-header-date")
```

Comment:
Let's not add the `id` to the component, since it is only used for testing -- you can get the same result from `wrapper.text()` or by exporting `HeaderText` and looking for that component.